sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/AbstractMethodParameterLookup.cs

```diff
@@ -70,6 +70,20 @@ public bool TryGetParameterSymbol(TArgumentSyntax argument, out IParameterSymbol
             return true;
         }
 
+        public bool TryGetSymbolParameter(IParameterSymbol parameter, out TArgumentSyntax argument)
```

Comment:
This method could also have a dedicated unit test for testing its actual behaviour